src/pks/flow/constitutive_relations/sources/pipe_drain_evaluator.hh

```diff
@@ -37,10 +38,14 @@ class PipeDrainEvaluator : public EvaluatorSecondaryMonotypeCV {
   virtual void EvaluatePartialDerivative_(const State& S,
           const Key& wrt_key, const Tag& wrt_tag,
           const std::vector<CompositeVector*>& result) override;
+  virtual void EnsureCompatibility_ToDeps_(State& S, const CompositeVectorSpace& fac) override;
```

Comment:
Please change name to `EnsureDepsCompatibility_`

Reply:
Hmm I don't think that would work? It is a virtual function.

Reply:
Sorry I thought it was a function you introduced